crypto/evp/evp_key.c

```diff
@@ -52,6 +52,7 @@ int EVP_read_pw_string(char *buf, int len, const char *prompt, int verify)
 int EVP_read_pw_string_min(char *buf, int min, int len, const char *prompt,
                            int verify)
 {
+#ifndef OPENSSL_NO_UI_CONSOLE
```

Comment:
Actually, this might not be the right change. I would rather check if `UI_get_default_method() == NULL` and then return -1.

Reply:
> UI_get_default_method

But it won't be null, because we are setting UI_null()

Reply:
@t8m fixed